hardhat.config.ts

```diff
@@ -80,6 +83,7 @@ const config: HardhatUserConfig = {
                     SEPOLIA_RETH_30DAY,
                     SEPOLIA_STETH_14DAY,
                     SEPOLIA_STETH_30DAY,
+                    SEPOLIA_MORPHO_BLUE_DAI_14DAY,
```

Comment:
Why do we need to update this config considering that we have a specific Sepolia config?

Reply:
I was using it to deploy everything on a forked hh network before going to sepolia (faster iteration times). Also want to have some base config so `npx hardhat` can be run in the root of the repo to show all commands. Happy to remove these bits or document somewhere if desired

Reply:
Gotcha, I just thought it was weird that we had sepolia here rather than mainnet. What I've been doing for testing is using the same network (in my case, it's usually been `NETWORK=mainnet`), but I'll point the RPC URL to a node that is a mainnet fork. Could you do something like that for testing? That may obviate the need for this.

Reply:
Been using it as a scratch file more or less so happy to remove the hyperdrive config from the file, however we still want this file to exist for reasons mentioned in other comments

Reply:
Yeah for sure. I was only referring to the inclusion of the sepolia yield sources FWIW. I know that we need `hardhat.config.ts`